public/app/plugins/panel/canvas/editor/TreeNavigationEditor.tsx

```diff
@@ -92,8 +141,27 @@ export const TreeNavigationEditor = ({ item }: StandardEditorProps<any, TreeView
         expandedKeys={expandedKeys}
         onExpand={onExpand}
         treeData={treeData}
+        titleRender={onTitleRender}
```

Comment:
This is getting really close! I'm noticing only two things at this point
1. A user is unable to select an element via the far left / right sides
2. There also is an interesting text color change when an element is selected and user hovers over action buttons

I think with these two things addressed we should be good to merge 💯 

https://user-images.githubusercontent.com/22381771/181033749-b08b0fee-54c1-45e2-b053-4d4c55dd4615.mov

Reply:
Updated 2, there was a styling from rc-tree interfering. For 1 - by default rc-tree allows selection by clicking on the node name - in our case the entire component for `titleRender`. Looking to see if there's anything to be done without interfering with other functionality. Imo it needs to be re-redesigned, it looks like a beaten up tree view..